### PR TITLE
Pass service as param to SEO integration test

### LINKED ISSUE #7247: Display Smart Banners on iOS for 3 services
**Is your feature request related to a problem? Please describe.**
For the three services with iOS apps, we should enable iOS to display a Smart Banner to promote use of our apps.

**Describe the solution you'd like**
In order to achieve this we need a meta tag in the head of the html document for each page on the relevant service, of the following form:

```
<meta name="apple-itunes-app" content="app-id=**********, app-argument=**********">
```

In the following table, canonicalUrl should be the canonicalUrl of the page the user is currently viewing. The values will most likely want to live in the service config.

Service | App-ID | App-Argument
------------ | ------------- | -------------
Russian | 504278066 | {canonicalUrl}?utm_medium=banner&amp;utm_content=apple-itunes-app
Mundo | 515255747 | {canonicalUrl}?utm_medium=banner&amp;utm_content=apple-itunes-app
Arabic | 558497376 | {canonicalUrl}?utm_medium=banner&amp;utm_content=apple-itunes-app

Note that the apps team have explicitly excepted front pages (and a number of other URLs) from showing the banner - these are listed in and controlled by https://www.bbc.com/.well-known/apple-app-site-association

**Acceptance criteria**

### Meta tag present
Given I am on a Simorgh page on Russian/Mundo/Arabic
Then the relevant meta tag is present

### Banner displayed with App not installed
Given I am using an iOS device
And I select a Simorgh page in safari on Russian, Mundo, or Arabic (live env only)
And the page URL is allowed in https://www.bbc.com/.well-known/apple-app-site-association
And I don't have the app installed
Then I can see a smart banner for the relevant App
And the CTA to install the app
And the correct Name and description for the app in my device locale

- Note the page url where we don't display it will be determined when a NOT is present e.g `"NOT /russian/in-depth-42173006",`

### Banner displayed with App installed
Given I am using an iOS device 
And I select a Simorgh page in safari on Russian, Mundo, or Arabic (live env only)
And the page URL is allowed in https://www.bbc.com/.well-known/apple-app-site-association
And I do have the app installed
Then I can see a smart banner for the relevant App
And the CTA to open the app
And the correct Name and description for the app in my device locale

### Banner not displayed on excluded pages (e.g. Front Pages)
Given I am using an iOS device
And I select a page in safari with the URL starting  https://www.bbc.com/russian
And the page URL is not allowed in https://www.bbc.com/.well-known/apple-app-site-association
And I don't have the app installed
Then I do not see the smart banner

### Select install
Given I can see the Smart Banner
And I do not have the app installed
When I select install
Then the app store opens
And I can see the relevant App store page

### Select open
Given I can see the Smart Banner
And I do have the app installed
When I select open
Then the correct app opens
And the page I was on opens

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #7247 

**Overall change:**
Pass `service` as a parameter to the SEO cross platform tests, so that it can be used for integration testing purposes
Refactor and tidy up affected integration tests

**Code changes:**
- Pass `service` as a parameter to the common cross platform tests and SEO tests
- Pass `service` as a parameter from all individual integration tests -> amp / canonical tests
- Ensure tests are consistent (`crossPlatformTests` should only be invoked from both `ampTests.js` & `canonicalTests.js`)
- Updating snapshots

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/integration/pages/articles/scotland/amp.test.js [hunk 1]
```diff
@@ -3,12 +3,10 @@
...
  */
 
-import runCrossPlatformTests from '../crossPlatformTests';
 import runAmpTests from '../ampTests';
 
 describe('AMP', () => {
   describe(pageType, () => {
-    runCrossPlatformTests('scotland');
-    runAmpTests();
+    runAmpTests(service);
```
Comment (chris-hinds) [inline src/integration/pages/articles/scotland/amp.test.js:10]: Where does `service` come from here?

**Review by chris-hinds — APPROVED** [review]

File: src/integration/pages/articles/scotland/amp.test.js (hunk 1 above)
Comment (karinathomasbbc) [reply src/integration/pages/articles/scotland/amp.test.js:10]: By declaring the `@service` in the header, it is then converted into a global var using docblock pragmas
https://github.com/bbc/simorgh/blob/latest/src/integration/integrationTestEnvironment.js#L14

Comment (karinathomasbbc) [reply src/integration/pages/articles/scotland/amp.test.js:10]: https://www.npmjs.com/package/jest-docblock

Comment (chris-hinds) [reply src/integration/pages/articles/scotland/amp.test.js:10]: Nice, thank you! Just realised this was in the README 👍

**Review by andrewscfc — APPROVED** [review]

**Comment by paruchurisilpa** [comment]
This doesn't require any manual testing.Moving to Ready to merge.

**Pull request merged**